### ISSUE TITLE
wildcard management in File system collector

### ISSUE THREAD

**Issue by mdemarqu** [ctron/rpm-builder#22]
It would be useful to handle wildcards or regex in file system collector in order to filter files easily.

example : 
```
<entry>
  <name>/usr/lib/foo</name>
  <collect>
    <from>target/*/*.so</from>
  </collect>
</entry>
```

**Comment by ctron** [comment]
I agree that would be quite useful. However I would try to re-use Mavens existing concept of `includes`/`excludes` like this:

~~~xml
<entry>
  <name>/usr/lib/foo</name>
  <collect>
    <from>target</from>
    <includes>
        <include>**/*.so</include>
    </includes>
    <excludes>
        <exclude>**/dont-use-me*.so</exclude>
    </excludes>
  </collect>
</entry>
~~~

**Comment by mdemarqu** [comment]
Yes, lot more elegant.

**Comment by Sami32** [comment]
I agree, more elegant and very useful in any case.

**Comment by ctron** [comment]
This should be fixed in version 1.0.0 which I just released. Please re-open otherwise.
